internals/secrethub/signup.go

```diff
@@ -158,6 +159,21 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	fmt.Fprintln(cmd.io.Stdout(), "Signup complete! You're now on SecretHub.")
-	fmt.Fprintf(cmd.io.Stdout(), "Please verify your email address to continue. We've sent a verification mail to %s\n", cmd.email)
+
+	fmt.Fprintln(cmd.io.Stdout(), "Setting up a start workspace.")
+
+	repoPath := secretpath.Join(cmd.username, "start")
+	_, err = client.Repos().Create(secretpath.Join(repoPath))
+	if err != nil {
+		return err
+	}
+
+	secretPath := secretpath.Join(repoPath, "hello")
+	_, err = client.Secrets().Write(secretPath, []byte("Welcome to SecretHub!"))
```

Comment:
Can we personalize this to use the user's `FullName`?

Reply:
And no ASCII art?

Reply:
How about this message:

```
Welcome Simon Barendse! This is your first secret. To write a new version of this secret, run:

   secrethub write simon/start/hello

```

Then the first two steps of the guide are totally self explanatory. You don't even strictly require docs for this.